motion_control.c

```diff
@@ -196,7 +196,7 @@ void mc_arc(float *position, float *target, float *offset, float radius, float f
 // Execute dwell in seconds.
 void mc_dwell(float seconds) 
 {
-  report_status_message(STATUS_OK); //report that we are dwelling
+  /* TODO: Report a dwell status? -JC */
```

Comment:
I know you have a note with a TODO here, but what is the reason for removing the report_status_message entirely?